packages/sui-test-contract/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@s-ui/test-contract",
+  "version": "1.0.0-beta.1",
+  "description": "Useful tooling for executing contract tests",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "main": "src/index.js",
+  "bin": {
+    "sui-test-contract": "bin/index.js"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lib": "babel --out-dir ./lib  ./src",
+    "prepare": "npm run lib"
+  },
+  "dependencies": {
+    "@pact-foundation/absolute-version": "0.0.4",
+    "@pact-foundation/pact": "9.16.4",
+    "commander": "6.2.1"
```

Comment:
Should we use v8.3.0 of commander? It is the same version as sui-bundler and some other tools.
![image](https://user-images.githubusercontent.com/5390428/154908808-db02be80-c05e-4ef4-85a7-ccaa238cb77f.png)